test/agentchat/test_conversable_agent.py

```diff
@@ -15,7 +15,7 @@ def conversable_agent():
 
 def test_trigger():
     agent = ConversableAgent("a0", max_consecutive_auto_reply=0, llm_config=False, human_input_mode="NEVER")
-    agent1 = ConversableAgent("a1", max_consecutive_auto_reply=0, human_input_mode="NEVER")
+    agent1 = ConversableAgent("a1", max_consecutive_auto_reply=0, llm_config=False, human_input_mode="NEVER")
```

Comment:
This might not be a blocking issue, but currently, the llm_config being set to "False" and "None" correspond to very different behaviors respectively. This might be something confusing or unconventional to some people. We need clear documentation.